mem-16-t.xml

```diff
@@ -7313,18 +7313,18 @@ This word was used for the "list" variable type in Klingon Blockly.[4]</column>
       <column name="definition_ru">тип струнного инструмента [AUTOTRANSLATED]</column>
       <column name="definition_zh_HK">弦樂器的類型 [AUTOTRANSLATED]</column>
       <column name="definition_pt">tipo de instrumento de cordas</column>
-      <column name="definition_fi">eräs kielisoitin</column>
+      <column name="definition_fi">eräs näppäiltävä kielisoitin</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{pang:v}, {yach:v:2}, {Heng:v}</column>
-      <column name="notes">This is a type of {HurDagh:n}.</column>
+      <column name="notes">This is a type of {HurDagh:n}. It is larger than {Supghew:n} and {leSpal:n}.</column>
```

Comment:
This is a good addition, but for consistency similar statements should also be added to **Supghew** and **leSpal**. (Also, it would be good to update the autotranslated notes to match the English text, if you can.)